contributing.md:62: [broken] https://pep8.org/: HTTPSConnectionPool(host='pep8.org', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000020CA3A13E90>: Failed to resolve 'pep8.org' ([Errno 11001] getaddrinfo failed)"))
changelog.md:5: [broken] https://keepachangelog.com/en/1.0.0/: HTTPSConnectionPool(host='keepachangelog.com', port=443): Max retries exceeded with url: /en/1.0.0/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000020CA3A10F10>: Failed to resolve 'keepachangelog.com' ([Errno 11001] getaddrinfo failed)"))
changelog.md:5: [broken] https://semver.org/spec/v2.0.0.html: HTTPSConnectionPool(host='semver.org', port=443): Max retries exceeded with url: /spec/v2.0.0.html (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000020CA3A13750>: Failed to resolve 'semver.org' ([Errno 11001] getaddrinfo failed)"))
